--- default-config/modules.conf
+++ adjusted-defaults/modules.conf
@@ -24,7 +24,7 @@
 ; the "require" keyword. Asterisk will exit with a status code of 2
 ; if a required module does not load.
 ;
-;require = chan_pjsip.so
+require = chan_pjsip.so
 ;
 ; If you want you can combine with preload
 ; preload-require = your_special_module.so
@@ -35,7 +35,7 @@
 ; By default, load chan_console only (automatically).
 ;
 noload = chan_alsa.so
-;noload = chan_console.so
+noload = chan_console.so
 ;
 ; Do not load res_hep and kin unless you are using HEP monitoring
 ; <http://sipcapture.org> in your network.
